Broken publish?

Seems like `trine/object/…` functions aren’t in the package:

```
Module not found: Error: Cannot resolve module 'trine/object/values'
```

See https://registry.npmjs.org/trine/-/trine-0.1.0.tgz – there’s no `object` folder there.

--- a/.gitignore
+++ b/.gitignore
@@ -4,5 +4,6 @@ node_modules
 /function
 /iterable
 /number
+/object
 /value
 /dist

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
     "function",
     "iterable",
     "number",
+    "object",
     "value"
   ],
   "dependencies": {

